hack/lib/etcd.sh

```diff
@@ -144,42 +144,46 @@ kube::etcd::cleanup() {
 }
 
 kube::etcd::install() {
-  local os
-  local arch
-
-  os=$(kube::util::host_os)
-  arch=$(kube::util::host_arch)
-
-  cd "${KUBE_ROOT}/third_party" || return 1
-  if [[ $(readlink etcd) == etcd-v${ETCD_VERSION}-${os}-* ]]; then
-    kube::log::info "etcd v${ETCD_VERSION} already installed. To use:"
-    kube::log::info "export PATH=\"$(pwd)/etcd:\${PATH}\""
-    # export into current process
-    PATH="$(pwd)/etcd:${PATH}"
-    export PATH
-    return  #already installed
-  fi
+  # Make sure that we will abort if the inner shell fails.
+  set -o errexit
+  set -o pipefail
+  set -o nounset
 
-  if [[ ${os} == "darwin" ]]; then
-    download_file="etcd-v${ETCD_VERSION}-${os}-${arch}.zip"
-    url="https://github.com/etcd-io/etcd/releases/download/v${ETCD_VERSION}/${download_file}"
-    kube::util::download_file "${url}" "${download_file}"
-    unzip -o "${download_file}"
-    ln -fns "etcd-v${ETCD_VERSION}-${os}-${arch}" etcd
-    rm "${download_file}"
-  elif [[ ${os} == "linux" ]]; then
-    url="https://github.com/etcd-io/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-${os}-${arch}.tar.gz"
-    download_file="etcd-v${ETCD_VERSION}-${os}-${arch}.tar.gz"
-    kube::util::download_file "${url}" "${download_file}"
-    tar xzf "${download_file}"
-    ln -fns "etcd-v${ETCD_VERSION}-${os}-${arch}" etcd
-    rm "${download_file}"
-  else
-    kube::log::info "${os} is NOT supported."
-  fi
-  kube::log::info "etcd v${ETCD_VERSION} installed. To use:"
-  kube::log::info "export PATH=\"$(pwd)/etcd:\${PATH}\""
```

Comment:
I find these logs useful, you removed  all of them

Reply:
a) They clutter the "normal path" screen with information I didn't ask for and usually don't need
b) They make it look like I need to do something (e.g. "cut and paste the `export`")
c) The third one actually tells me to do exactly that

How about I V=3 them?

Reply:
Actually, cut and paste is needed if you want to execute hack/local-up-cluster.sh after this.

If you want to build a local cluster,
1. ./hack/install-etcd.sh
2. export PATH=to this PATH
3. ./hack/local-up-cluster.sh